airbyte-webapp/src/services/connectorBuilder/connector_manifest_openapi.yaml

```diff
@@ -0,0 +1,9 @@
+openapi: 3.0.0
+info:
+  title: Connector Manifest schema
+  version: 1.0.0
+paths: {}
+components:
+  schemas:
+    ConnectorManifest:
+      $ref: "../../../../airbyte-cdk/python/airbyte_cdk/sources/declarative/config_component_schema.json"
```

Comment:
FYI I added this file so that we could use orval to pull in the Connector Manifest json schema and generate typescript types from it. This is necessary to do rather than referencing the schema directly in the connector builder server openapi spec, because of the issue described here: https://github.com/airbytehq/airbyte/issues/20248